src/shared/test/multiline-strings-test.js

```diff
@@ -0,0 +1,32 @@
+import { trimAndDedent } from '../multiline-strings';
+
+describe('trimAndDedent', () => {
+  [
+    ['Foo', 'Foo'],
```

Comment:
This could use a test for an indented first line. If you don't need to preserve the indentation of the first/last lines you could just use `str.trim()`.